.github/actions/specs-report/__tests__/mocks.ts

```diff
@@ -0,0 +1,272 @@
+export const TBDEX_TEST_VECTORS_FILES = [
```

Comment:
in the future could dynamically get them from tbdex submodule incase a new one is added?

Reply:
the purpose of these mocks is to test the action itself, you dont need to keep updating them. it was just some test data to make sure the unit tests for the action is passing